spec/run_spec.cr

```diff
@@ -0,0 +1,84 @@
+require "./spec_helper"
+
+describe Run do
+  describe "Load expect data" do
+    it "should return error message when pass invalid argument" do
+    end
```

Comment:
@lanjoni Here I have doubts about how we could test invalid messages without passing an IO as an argument to the functions,

Reply:
But I think it's not worth the effort to test the invalid message is appearing correctly.

Reply:
It may not be necessary to validate IO for the tests we are adding here! I believe that the primary tests that we can validate are:
- Are flags working correctly and as they should?
- Are variables being used internally as expected?
- Conditional check to know if a test passed or is not working correctly?

Another point would be the addition of the `-d` flag, which will be used to upload a Docker container of a certain programming language to perform the tests, so perhaps it is one of the points to validate.

But so far everything is excellent! Congratulations in advance!

Reply:
@igorwessel @lanjoni Hey! Maybe testing if the control flow is running correctly with different user based interactions would be an option?

For example, when the user uses the `-i` flag, there should be a filename just after it. So, if we ran hackacrow while pushing no value to the `-i` flag we should at least expect the inner expect file path to be different from the default one. Any other user based error is out of the current spec necessities.

Anyways, I agree with @lanjoni by not needing (at least for now) to rely on testing IO itself right now.